cmd/utils/nodecmd/accountcmd_test.go

```diff
@@ -46,7 +46,8 @@ func tmpDatadirWithKeystore(t *testing.T) string {
 }
 
 func TestAccountListEmpty(t *testing.T) {
-	klay := runKlay(t, "klay-test", "account", "list")
+	datadir := tmpdir(t)
+	klay := runKlay(t, "klay-test", "account", "list", "--datadir", datadir)
```

Comment:
I think datadir doesn't need to be set here. The datadir is automatically set in runKlay func, right?

Reply:
You're right. It got fixed by MigrateGlobalFlags. Reverted to original at 1929acc1.